lib/systems/platforms.nix

```diff
@@ -433,6 +433,24 @@ rec {
     };
   };
 
+  mips64el-multiplatform = {
+    name = "mips64el-multiplatform";
+    kernelArch = "mips";
+    kernelTarget = "vmlinux";
+    kernelAutoModules = true;
+#   kernelBaseConfig = "cavium_octeon_defconfig";
```

Comment:
Do you know if there's a multiplatform config that works on both the edgerouter and QEMU?

Reply:
Yeah it should be big-endian - I actually looked at some point for what are the options for purchasing big-endian MIPS hardware and EdgeRouter Lite was the only even close to reasonable option to buy...